sycl/include/sycl/sycl.hpp

```diff
@@ -0,0 +1,11 @@
+//==------------ sycl.hpp - SYCL2020 standard header file ------------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include <CL/sycl.hpp>
```

Comment:
you have changed only single file. Isn't it reasonable to modify all except one which will check legacy mode?

Reply:
Could you please clarify if you suggest to modify implementation to use `sycl/` instead of `CL/` or tests?

Reply:
I was speaking about tests, but it is NIT change, so fill free to skip it for now.

Reply:
Ok, I would like to avoid massive changes in tests without added value.